### PR TITLE
Add extra points to pr template

### PR DESCRIPTION
**Pull request by tomdye**
Add's extra points to the PR template to ensure that widget variants use `theme.compose` and `WidgetProperties` are exported to enable document generation.
Also, removes point re. `functional tests` as we no longer have those.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -4,7 +4,9 @@ The following has been addressed in the PR:
 
 * [ ] There is a related issue
 * [ ] All code matches the [style guide](https://github.com/dojo/framework/blob/master/STYLE.md)
-* [ ] Unit or Functional tests are included in the PR
+* [ ] Unit tests are included in the PR
+* [ ] Any widget variant uses `theme.compose`
```
Comment (agubler) [inline .github/PULL_REQUEST_TEMPLATE.md]: Any chance you could link this to either an example or the issue that describes the pattern?

**Comment by agubler** [comment]
Can we add something about the checks on the documentation, to ensure that properties and theme classes are rendered?

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* dojo.widgets: https://dojowidgets-h1lxzn3sn.now.sh

**Comment by agubler** [comment]
@tomdye do you think you could address the comment and land?

**Comment by tomdye** [comment]
comments addressed @agubler

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* Docs Deployment: https://dojowidgets-9obgytzhr.now.sh
* Tests Deployment: https://dojowidgets-ip6ldbljw.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* Docs Deployment: https://dojowidgets-9obgytzhr.now.sh
* Tests Deployment: https://dojowidgets-ip6ldbljw.now.sh

**Pull request merged**